e2e/tests/admin/user-admin-ui.spec.ts

```diff
@@ -0,0 +1,99 @@
+import UserAdminPage from 'app/page/admin-user-list-page';
+import { signInWithAccessToken } from 'utils/test-utils';
+import navigation, { NavLink } from 'app/component/navigation';
+import AdminTable from 'app/component/admin-table';
+import UserProfileInfo from 'app/page/admin-user-profile-info';
+//import UserAuditPage from 'app/page/admin-user-audit-page';
+
+
+describe('Admin', () => {
+  const userEmail = 'admin_test@fake-research-aou.org';
+  // const userName = "admin_test";
+
+  beforeEach(async () => {
+    await signInWithAccessToken(page);
+    //await navigation.navMenu(page, NavLink.ADMIN);
+    await navigation.navMenu(page, NavLink.USER_ADMIN);
+  });
+
+  test('verify user bypassed and status is active', async () => {
+    const userAdminPage = new UserAdminPage(page);
+    await userAdminPage.waitForLoad();
+
+    //look up for the user
+    await userAdminPage.searchUser(userEmail);
+    await userAdminPage.waitForLoad();
+
+    //Verify table column names match.
+    const columns = [
+      'Status',
+      'Institution',
+      'Registration date',
+      'User name',
+      'Contact Email',
+      'User Lockout',
+      'First Sign-in',
+      '2FA',
+      'Training',
+      'eRA Commons',
+      'DUCC',
+      'Bypass',
+      'Audit'
+    ];
+
+    const adminTable = new AdminTable(page);
+    const columnNames = await adminTable.getColumnNames();
+    console.log(columnNames);
+    expect(columnNames).toHaveLength(columns.length);
+    expect(columnNames.sort()).toEqual(columns.sort());
+
+    // get the index of the Bypass column
+    const bypassColIndex = await adminTable.getColumnIndex('Bypass');
+
+    //click on the bypass link
+    const bypassLinkModal = await userAdminPage.clickBypassLink(1, bypassColIndex);
+    const toggleText = await bypassLinkModal.getAllToggleTexts();
+    console.log(toggleText);
+
+    // verify all the 4 modules toggle are checked
+    await bypassLinkModal.verifyAllModulesBypassed();
+
+    const user1UserLockout = 'Disable';
```

Comment:
This is brittle. Test breaks when the first value is `Active`.
Suggestion: a better test workflow could be:
Switch: 
   When Status === Disabled, then User Lockout === Enable. Break;
   When Status === Active, then User Lockout === Disable. Break;

Reply:
Thanks, will refactor!